redfish-core/lib/processor.hpp

```diff
@@ -1100,13 +1100,15 @@ inline void requestRoutesOperatingConfigCollection(App& app)
 
```

Comment:
There are more changes in https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/network_protocol.hpp#326

Reply:
Please add a line number where changes are missed.

Reply:
Sorry, this is for network_protocol.hpp